workflows/photovoltaic.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Photovoltaic Workflows
+parent: Workflows
+nav_order: 2
+---
+
+# Photovoltaic Workflows
+
+As of version 0.6.3, URBANopt supports community photovoltaic and ground mount photovoltaic workflows. 
+
+## Community Photovoltaic
+
+Community Photovoltaic features can be added to the GeoJSON Feature File as a district system of type "Community Photovoltaic".  View the [district system schema](https://docs.urbanopt.net/urbanopt-geojson-gem/schemas/district-system-properties.html) for more details. The REopt scenario optimization post-processor will use this information in its PV capacity calculations. 
+
```

Comment:
@kflemin Should we add something here to say that : 'the REopt scenario optimization post-processor will use this information in its PV capacity calculations, when run with the multi-PV assumption file'

Reply:
good call, thanks!